examples/unix-seqpacket-server.rs

```diff
@@ -10,7 +10,7 @@ struct Options {
 	socket: PathBuf,
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Will it have performance impacts, running the server in a single thread?

Reply:
Yes, although it will be minimal. In this case though, this was already the default for tokio 0.2, so it's not really a change, just more explicit.

Reply:
The reason for using a single threaded runtime here is to avoid pulling in the multi-threading code in tokio for the examples.
Normally, the I/O part of a server also isn't very CPU intensive, as long as computationally heavy tasks are executed in a different thread.

With C++, we also stick with a single threaded executor for I/O to avoid race conditions. In Rust, that is not really a concern anymore though, since it doesn't let us write data races that easily.